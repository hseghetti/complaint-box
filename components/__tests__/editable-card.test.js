import React from 'react';
import renderer from 'react-test-renderer';
import EditableCard from '../editable-card';

describe('Card Component', () => {
  it('Should render Card', () => {
    const component = renderer.create(
      <EditableCard />
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});