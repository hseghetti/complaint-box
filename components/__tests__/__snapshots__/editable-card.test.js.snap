// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component Should render Card 1`] = `
<div
  className="css-rla46g"
>
  <button
    aria-controls="popover-content-2"
    aria-expanded={false}
    aria-haspopup="dialog"
    className="chakra-button css-ujq32"
    id="popover-trigger-2"
    onClick={[Function]}
    type="button"
  >
    <svg
      aria-hidden={true}
      className="chakra-icon css-11up8ym"
      focusable={false}
      viewBox="0 0 24 24"
    >
      <path
        d="M.439,21.44a1.5,1.5,0,0,0,2.122,2.121L11.823,14.3a.25.25,0,0,1,.354,0l9.262,9.263a1.5,1.5,0,1,0,2.122-2.121L14.3,12.177a.25.25,0,0,1,0-.354l9.263-9.262A1.5,1.5,0,0,0,21.439.44L12.177,9.7a.25.25,0,0,1-.354,0L2.561.44A1.5,1.5,0,0,0,.439,2.561L9.7,11.823a.25.25,0,0,1,0,.354Z"
        fill="currentColor"
      />
    </svg>
  </button>
  <section
    aria-describedby="popover-body-2"
    aria-labelledby="popover-header-2"
    className="chakra-popover__content css-17cduhr"
    hidden={true}
    id="popover-content-2"
    onKeyDown={[Function]}
    role="dialog"
    style={
      Object {
        "transformOrigin": "top center",
      }
    }
    tabIndex={-1}
  >
    <div
      className="chakra-popover__arrow css-vro3gk"
      style={
        Object {
          "boxShadow": "-1px -1px 1px 0 whiteAlpha.300",
          "height": 10,
          "position": "absolute",
          "top": "-5px",
          "transform": "rotate(45deg)",
          "width": 10,
        }
      }
    />
    <button
      aria-label="Close"
      className="css-15k60nl"
      onClick={[Function]}
      type="button"
    >
      <svg
        aria-hidden={true}
        className="chakra-icon css-onkibi"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M.439,21.44a1.5,1.5,0,0,0,2.122,2.121L11.823,14.3a.25.25,0,0,1,.354,0l9.262,9.263a1.5,1.5,0,1,0,2.122-2.121L14.3,12.177a.25.25,0,0,1,0-.354l9.263-9.262A1.5,1.5,0,0,0,21.439.44L12.177,9.7a.25.25,0,0,1-.354,0L2.561.44A1.5,1.5,0,0,0,.439,2.561L9.7,11.823a.25.25,0,0,1,0,.354Z"
          fill="currentColor"
        />
      </svg>
    </button>
    <header
      className="chakra-popover__header css-0"
      id="popover-header-2"
    >
      Confirmation
    </header>
    <div
      className="chakra-popover__body css-0"
      id="popover-body-2"
    >
      Do you really want to delete the comment?
      <button
        className="chakra-button css-1y5ccfj"
        onClick={[Function]}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="chakra-icon css-dvq678"
          focusable={false}
          viewBox="0 0 24 24"
        >
          <path
            d="M12,0A12,12,0,1,0,24,12,12.014,12.014,0,0,0,12,0Zm6.927,8.2-6.845,9.289a1.011,1.011,0,0,1-1.43.188L5.764,13.769a1,1,0,1,1,1.25-1.562l4.076,3.261,6.227-8.451A1,1,0,1,1,18.927,8.2Z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
  </section>
  <textarea
    aria-readonly={true}
    className="chakra-textarea css-5d6l6w"
    onBlur={[Function]}
    onFocus={[Function]}
    readOnly={true}
  />
</div>
`;
